docs/deployment.md

```diff
@@ -4,9 +4,8 @@ The apply build and release process is split into two separate GitHub Actions pi
 
 - [Build](https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/build.yml): This is the main CI pipeline which will automatically trigger a build from a commit to the `main` branch, this pipeline is also run on the merged PR head commit when a pull request is raised targetting `main`. If run from `main` branch this also triggers the deployment pipeline after all tests have succeeded.
 
-- [Deploy](https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/deploy.yml): This is the main release pipeline that is used to deploy to all environments. Releases to `production` are triggered manually and the target environments can be chosen prior to deployment.
+- [Deploy](https://github.com/DFE-Digital/apply-for-teacher-training/actions/workflows/deploy.yml): This is the main release pipeline that is used to deploy to all environments. Continuous Deployment is enabled so a successful build on `main` triggers deployments to all environments including `production`.  Deployments can be triggered manually and the target environments can be chosen prior to deployment, see the [Manual Deployment guide](manual-deployment.md) for instructions.
```

Comment:
Can we mention the review apps? They are essential for CD

Reply:
I've added an explanation of that process